apps/modia-soknadsstatus-api/src/main/kotlin/no/nav/modia/soknadsstatus/service/HendelseService.kt

```diff
@@ -112,6 +141,23 @@ class HendelseServiceImpl(
             ansvarligEnhet = hendelse.ansvarligEnhet,
         )
 
+    private fun behandlingToHendelseDAO(
+        modiaBehandlingId: String,
+        behandling: InnkommendeBehandling,
+    ): SoknadsstatusDomain.Hendelse =
+        SoknadsstatusDomain.Hendelse(
+            modiaBehandlingId = modiaBehandlingId,
+            hendelseId = behandling.behandlingId,
+            behandlingId = behandling.behandlingId,
```

Comment:
Vi vil vel unngå samme ID her?

Reply:
this is ok in my opinion since we have only one hendelse per behandling